pyneuroml/plot/PlotMorphologyVispy.py

```diff
@@ -356,6 +373,11 @@ def plot_interactive_3D(
         If you have a good GPU, you can increase both these values to get more
         detailed visualizations
     :type precision: (int, int)
+    :param upright: bool only applicable for single cells: Makes cells "upright"
+    (along Y axis) by calculating it PCA and transforming cell co-ordinates to
```

Comment:
I *think* multiline docstrings require the next line to be indented (check the rtd CI task to see what it looks like to be sure)

Reply:
Hm, changed it but it still shows the first line only. This happens to the other parameters with multiple lines too however..

Reply:
OK, let's leave it for the moment. I can take a look at it after we merge and see what's going on.